crates/bevy_ecs/src/system/system_piping.rs

```diff
@@ -121,33 +122,34 @@ impl<SystemA: System, SystemB: System<In = SystemA::Out>> System for ChainSystem
     }
 }
 
-/// An extension trait providing the [`IntoChainSystem::chain`] method for convenient [`System`]
-/// chaining.
+/// An extension trait providing the [`IntoPipeSystem::pipe`] method to pass input from one system into the next.
+///
+/// The first system must have a non-empty return type `T`
```

Comment:
```suggestion
/// The first system must have return type `T`
```

It's technically possible to pipe `()`, no?

Reply:
True, but not very useful. You're correct though.